gnovm/stdlibs/testing/random_test.gno

```diff
@@ -0,0 +1,101 @@
+package testing
+
+import (
+	"math"
+	"strconv"
+	"time"
+)
+
+func updateSeed() {
+	seed := time.Now().UnixNano()
+	_srand(seed)
+}
+
+// TODO: use chi-squared test to check number distribution follows uniform distribution.
+func Test_UniformRand(t *T) {
+	valueMap := make(map[uint64]int)
+	maxIter := 1000
+
+	for i := 0; i < maxIter; i++ {
+		result := UniformRand()
+
+		if result < 0 || result >= 32768 {
+			t.Errorf("rand() = %v, want in range [0, 32767]", result)
+		}
+
+		if _, ok := valueMap[result]; ok {
+			valueMap[result]++
+		} else {
+			valueMap[result] = 1
+		}
+	}
+
+	lvm := len(valueMap)
+	if lvm > maxIter || lvm == 0 {
+		t.Errorf("len(valueMap) = %v, want <= %v", lvm, maxIter)
+	}
+}
+
+// TODO: use chi-squared test to check number distribution follows normal distribution.
```

Comment:
Leftover?

Reply:
It's necessary for a more accurate test. but it also seems there's no need to implement anything extra just for the test. so yes leftover, I think so.